docs/immortal-empires/norsca.md

```diff
@@ -25,7 +25,7 @@
 	Skavenblight, Khemri, The Awakening, Wei-Jin, Zhar-Naggrund
   * **Reward**:
 	* _The Eternal Challenger_: 
-	  * Leadership: -20% for enemy chracters near Wulfrik, give him Grimgor or Snitch ability
+	  * Ability: "Hunter of Champions"

```

Comment:
Ability: "Hunter of Champions" (maybe a short description, optional) **for Wulfrik**